leveldb/memdb/memdb.go

```diff
@@ -469,11 +478,31 @@ func (p *DB) Reset() {
 func New(cmp comparer.BasicComparer, capacity int) *DB {
 	p := &DB{
 		cmp:       cmp,
+		quickCmp:  (cmp == comparer.DefaultComparer),
```

Comment:
It's wrong here. Actually the cmp used in leveldb is `iComparer` with the notion of `sequence`.
It means the `quickCmp` is disabled by default.